### ISSUE TITLE
No matching export in "node-modules-polyfills:url" for import "URL"

### ISSUE THREAD

**Issue by binajmen** [mollie/mollie-api-node#268]
Hi,

I'm using Remix (https://remix.run) with Mollie. I have this error while building my app.

Not sure to understand where the error come from. Are you relying on polyfills I should add to my dependencies?

```ts
[1] ✘ [ERROR] No matching export in "node-modules-polyfills:url" for import "URL"
[1] 
[1]     node_modules/@mollie/api-client/dist/mollie.esm.js:5:9:
[1]       5 │ import { URL } from 'url';
[1]         ╵          ~~~
[1] 
[1] 
[1] Build failed with 1 error:
[1] node_modules/@mollie/api-client/dist/mollie.esm.js:5:9: ERROR: No matching export in "node-modules-polyfills:url" for import "URL"
```

Your help would be greatly appreciated 🙏

**Comment by Pimm** [comment]
This library doesn't rely on any polyfills and only depends on axios, which NPM (or Yarn) will automatically install for you.

The current version runs on Node.js 6.14.×, which has [`url`](//nodejs.org/api/url.html) available.

**Comment by binajmen** [comment]
> This library doesn't rely on any polyfills and only depends on axios, which NPM (or Yarn) will automatically install for you.
> 
> The current version runs on Node.js 6.14.×, which has [`url`](//nodejs.org/api/url.html) available.

Yes it was totally my fault ;)

**Comment by dodas** [comment]
Hey there!
I'm running into the same issue (using node 14.17.3).
Mind sharing how you solved it @binajmen ?
Thanks.

**Comment by binajmen** [comment]
> Hey there! I'm running into the same issue (using node 14.17.3). Mind sharing how you solved it @binajmen ? Thanks.

Hi @dodas, this problem was specific to how Remix works. If you're using Remix, you have to add the suffix `.server.ts` to keep things at server side.

**Comment by dodas** [comment]
Yes, I am using Remix. I managed to fix it in a similar manner – the package being bundled into code frontend was the problem.

**Comment by Pimm** [comment]
Thanks for commenting. Chances are that you're helping out the next person who finds themself in the same situation.

I'm updating my first answer to be a little more helpful.
